sdk/identity/azure-identity/inc/azure/identity/workload_identity_credential.hpp

```diff
@@ -61,42 +80,29 @@ namespace Azure { namespace Identity {
     std::string m_tokenFilePath;
 
     explicit WorkloadIdentityCredential(
-        std::string tenantId,
-        std::string const& clientId,
-        std::string const& tokenFilePath,
-        std::string const& authorityHost,
+        std::string const& tenantIdOptions,
```

Comment:
Naming a variable `xxxOptions` implies that there are more than one possible value for the field. There should be comments describing the possible options contained in each of `tenantIdOptions`, `clientIdOptions`, etc.

Reply:
The naming here isn't the best and meant only to differentiate between the values passed-in from the options bag vs locals (rather than about a set of possible options to pick from). That said, this is a private ctor.

Reply:
IMHO, It shouldn't matter if the ctor is private or not. Naming is naming, whether it's public or private.

Why do we need this ctor? There doesn't appear to be any comments explaining why we need a 5 parameter private ctor.